message_adapter/message_adapter.py

```diff
@@ -6,14 +6,16 @@
 import uuid
 from jsonpath_ng import parse
 from jsonschema import validate
+from collections import defaultdict
+from copy import deepcopy
 from .aws import stepFn, s3
 
+
 class message_adapter:
     """
     transforms the cumulus message
     """
-    # Maximum message payload size that will NOT be stored in S3. Anything bigger will be.
-    MAX_NON_S3_PAYLOAD_SIZE = 10000
+    REMOTE_DEFAULT_MAX_SIZE = 10000
```

Comment:
Is this still relevant with the manual configuration? Are we _not_ storing it on s3 when the size is below this number, despite the user configuring it to be stored on S3? Seems to just add confusing conditionals, is that worthwhile?

Reply:
It's mostly to retain compatibility with existing workflows/expectations.   If you configure a replacement config but *don't* set the value you need a default behavior.  We could set it to zero, or fail instead.

Reply:
Configurability is nice for debugging, in theory, and some users may want to not bother with variability in their messages, or some may prefer to have it available without hitting S3 if at all possible.        As we're targeting a portion of the message and putting a size constraint on that, versus the size of the entire message, that seems tunable. 

In this instance I wasn't sure what user needs would be, so opted for configurability over a lack of it.   Perhaps the wrong approach?

We could still use the size of the entire message, which would make the useful configurable settings to be '0' and <max message size> with marginal possible usefulness for debugging at some value just high enough above zero to allow a test case or something.

Reply:
Setting default value to 0 : fabfd50

Reply:
also 75f1fc5